### PR TITLE
I330 Release notes for 2.1. release

### PR DESCRIPTION
**Pull request by Tereza-Pytelova**

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
+## Release 1.5.2019 v2.1
+WAW was moved to Python 3.5 and from this release it is not compatible with Python 2. Last WAW version which supports python 2.7 is the 1.2.
+
+### New Features
+ - You can now delete, upload and test workspace by specifying its name or regular expression
+ - It is possible to replace some values using [workspace_addjson](/scripts/workspace_addjson.py) script
+ - There is now an option to set the log level in the CLI.
+ - `-v` flag now just print more INFO messages (not DEBUG or ERROR).
+ - Documentation and tests for actions in the dialog was added.
```
Comment (ota-sandr) [inline release_notes.md]:
```suggestion
 - Documentation and tests for actions in the dialog were added.
```

File: release_notes.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+
+#### Cloud Functions
+ - While using cloud functions is is now possible to use an apikey
```
Comment (ota-sandr) [inline release_notes.md]:
```suggestion
 - While using cloud functions, it is now possible to use an apikey
```

File: release_notes.md [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+ - Script [functions_delete_package](/scripts/functions_delete_package.py) for deleting cf package was added
+ - Script [functions_deploy](/scripts/functions_deploy.py) is now able to deploy also CF sequences
+
+#### Testing
+ - If you set `exception_if_fail` parameter of script [evaluate_tests](/scripts/evaluate_tests.py) to true, it throws exception, when there are any failed tests detected while evaluating.
+ - From now we are running tests for Python 3 only
```
Comment (ota-sandr) [inline release_notes.md]:
```suggestion
 - We are running tests for Python 3 only from now
```

File: release_notes.md [hunk 4]
```diff
@@ -1,4 +1,24 @@
+## Release 1.5.2019 v2.1
+WAW was moved to Python 3.5 and from this release it is not compatible with Python 2. Last WAW version which supports python 2.7 is the 1.2.
+
+### New Features
+ - You can now delete, upload and test workspace by specifying its name or regular expression
+ - It is possible to replace some values using [workspace_addjson](/scripts/workspace_addjson.py) script
+ - There is now an option to set the log level in the CLI.
```
Comment (ota-sandr) [inline release_notes.md]: Some bullets have period on the end and some not. They should be all same.

File: release_notes.md [hunk 5]
```diff
@@ -1,4 +1,24 @@
+## Release 1.5.2019 v2.1
+WAW was moved to Python 3.5 and from this release it is not compatible with Python 2. Last WAW version which supports python 2.7 is the 1.2.
```
Comment (ota-sandr) [inline release_notes.md]:
```suggestion
WAW was moved to Python 3.5 and it is not compatible with Python 2 from this release. Last WAW version which supports python 2.7 is the 1.2.
```

File: release_notes.md (hunk 4 above)
Comment (Tereza-Pytelova) [reply release_notes.md]: Removed.

File: release_notes.md (hunk 3 above)
Comment (Tereza-Pytelova) [reply release_notes.md]: Rewritten.

File: release_notes.md (hunk 5 above)
Comment (Tereza-Pytelova) [reply release_notes.md]: Rewritten.

**Pull request merged**